src/utils/index.ts

```diff
@@ -1355,10 +1355,16 @@ export function toIdentityWithClaimsArray(
 /**
  * @hidden
  */
-export function meshProposalToCall(bytes: string, context: Context): Call {
-  const call = context.polymeshApi.createType('Proposal', bytes);
-  return {
-    method: call.methodName,
-    module: call.sectionName,
-  };
+export function middlewareProposalToTxTag(bytes: string, context: Context): TxTag {
+  const proposal = context.polymeshApi.createType('Proposal', bytes);
+  const { sectionName, methodName } = proposal;
+
+  let moduleName;
+  for (const txTagItem in TxTags) {
+    if (txTagItem.toLowerCase() === sectionName.toLowerCase()) {
+      moduleName = txTagItem;
+    }
+  }
+
+  return `${moduleName}.${camelCase(methodName)}` as TxTag;
```

Comment:
Why copy/paste the code from `extrinsicIdentifierToTxTag` when you can just call the function?

Reply:
(Just remember to use .toLowerCase on the original's moduleName)